docs/source/models/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -3272,15 +3272,16 @@ Limitations:
   If the process of determining elgibility prior to selecting simulants for noise is challenging, we can work on finding a simpler approach. 
 
 
-**Numeric Miswriting** 
+**Numeric Miswriting**
 
-To implement this, select the strings eligible for noise and apply 
-the numeric miswriting noise function to all strings with the user defined 
-character error rate. 
+To implement this, select the strings eligible for noise,
+and then select, for each each digit character (0-9) in each eligible string, whether to change that character, according to the per-character error probability.
+The digits that are changed should be changed to a *different* digit character, selected uniformly at random.
```

Comment:
Requiring a different character seems like the same problem I told the engineers not to worry about yesterday: requiring a new selection with incorrect select. But maybe since we have a standard pre-defined set of choices [0,1,...,9], it will be easier to implement?

Reply:
Yes, good point. I'll add a note that this is a nice-to-have.